layers/!lang/ruby/README.org

```diff
@@ -87,3 +90,14 @@ a couple of useful keybindings:
 |-------------+---------------------|
 | ~SPC m t b~ | run test file       |
 | ~SPC m t t~ | run test at pointer |
+
+** rubocop
+
+| Key binding   | Description                   |
+|---------------+-------------------------------|
+| ~SPC m r a f~ | check current file            |
+| ~SPC m r a d~ | check current directory       |
+| ~SPC m r a p~ | check current project         |
```

Comment:
I was opposed to #2362 on these grounds, but literally the only saving grace for this is the autocorrect functionality...I agree with @syl20bnr on this one...leave the autocorrect stuff though, that's handy.